tiledb/sm/query/query.cc

```diff
@@ -670,7 +670,7 @@ Status Query::get_data_buffer(
   }
 
   // Check attribute
-  if (name != constants::coords) {
+  if (name != constants::coords && name != constants::timestamps) {
```

Comment:
No, this is an existing case. I needed to add this check as a fix because `get_data_buffer` is called in `query_from_capnp` and it was failing in case of `name == constants::timestamps` without it.
Calling @kiterluc to decide if that fix looks right or I am just covering up another bug of mine. In other words is it normal that buffers with timestamps are deserialized when a query is deserialized?

Reply:
Use ArraySchema::is_special_attribute

Reply:
We should also fix Query::get_buffer with the same check.